### ISSUE TITLE
Add ability to change user's group

### ISSUE THREAD

**Issue by steven-basart** [centerforaisafety/cerberus-cluster#90]
Add the following function to cluster file:
```
...
from ldap3 import MODIFY_ADD, MODIFY_DELETE
...

@group.command()
@click.argument('curr_group')
@click.argument('new_group')
@click.argument('user')
def change_group(curr_group, new_group, user):
    """
      Change user's group.
      Parameters:
        curr_group (group Name)
        new_group (Group Name)
        user (User Name)
    """
    server = ldap3.Server(host, use_ssl=True)
    with ldap3.Connection(server, bind_dn, bind_pass, auto_bind=True) as conn:
        # Add the user to the new group
        conn.modify("cn={},{}".format(new_group, groups_dn),
                    {'member': [(MODIFY_ADD, ["cn={},{}".format(user, people_dn)])]})

        # Remove the user from the current group
        conn.modify("cn={},{}".format(curr_group, groups_dn),
                    {'member': [(MODIFY_DELETE, ["cn={},{}".format(user, people_dn)])]})

    print(conn.result)
```

**Comment by steven-basart** [comment]
The above didn't work unfortunately :(

**Comment by steven-basart** [comment]
Create two groups
```
cluster group create group1
{'result': 0, 'description': 'success', 'dn': '', 'message': '', 'referrals': None, 'type': 'addResponse'}
[opc@dev-cpu-cluster-bastion bin]$ cluster group create group2
{'result': 0, 'description': 'success', 'dn': '', 'message': '', 'referrals': None, 'type': 'addResponse'}
```

Confirm the groups are created.
```
cluster group list
DN: cn=group1,ou=Group,dc=local - STATUS: Read - READ TIME: 2023-05-24T23:55:50.905555
    cn: group1
    gidNumber: 10006
    objectClass: top
                 groupOfMembers
                 posixGroup

DN: cn=group2,ou=Group,dc=local - STATUS: Read - READ TIME: 2023-05-24T23:55:50.905698
    cn: group2
    gidNumber: 10007
    objectClass: top
                 groupOfMembers
                 posixGroup
```

Add a user to group1
```
cluster user add test_user --gid 10006
Password: 
Repeat for confirmation: 
Full Name: Test User
```

Confirm that they've been made
```
cluster user list
DN: cn=test_user,ou=People,dc=local - STATUS: Read - READ TIME: 2023-05-25T00:11:22.292787
    cn: test_user
    displayName: Test User
    gecos: Test User
    gidNumber: 10006
    homeDirectory: /data/test_user
    loginShell: /bin/bash
    sn: test_user
    uid: test_user
    uidNumber: 10001
```

gidNumber is correct.  Okay now let's try to switch it.

```
cluster group change_group group1 group2 test_user
{'result': 20, 'description': 'attributeOrValueExists', 'dn': '', 'message': 'modify/add: member: value #0 already exists', 'referrals': None, 'type': 'modifyResponse'}
[opc@dev-cpu-cluster-bastion bin]$ cluster user list
DN: cn=test_user,ou=People,dc=local - STATUS: Read - READ TIME: 2023-05-25T00:18:58.192679
    cn: test_user
    displayName: Test User
    gecos: Test User
    gidNumber: 10006
    homeDirectory: /data/test_user
    loginShell: /bin/bash
    sn: test_user
    uid: test_user
    uidNumber: 10001
```
Not changed.

I've also tried it with remove first then add to new group in cluster file.

**Comment by steven-basart** [comment]
Fixed in PR #102
